docs/source/installation/index.rst

```diff
@@ -172,6 +172,37 @@ After updating your ``PATH`` you can check if it worked in the following way:
   environment variable in your ``.bashrc``::
   
     export PYTHONPATH="${PYTHONPATH}:<AiiDA_folder>"
+
+
+Using AiiDA in Jupyter
+++++++++++++++++++++++
+
+`Jupyter <http://jupyter.org>`_ is an open-source web application that allows you to create in-browser notebooks containing live code, visualizations and formatted text.
+
+Originally born out of the iPython project, it now supports code written in many languages and customized iPython kernels.
+
+To test Jupyter, make sure you are have installed AiiDA with the ``notebook`` option, activate the AiiDA virtual environment and run::
```

Comment:
I'm not sure it's super clear. I would rather say

  If you didn't already install AiiDA with the ``[notebook]`` option (during ``pip install``), run ``pip install jupyter`` **inside** the virtualenv, and then run **from within the virtualenv**::